web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -141,7 +140,10 @@
                 </v-btn>
               </div>
             </template>
-            {{ publishDisabledMessage }}
+            <p
+              :style="{ whiteSpace: 'pre-line' }"
```

Comment:
I have been told that generally you put this stuff into a CSS class in a separate CSS block.
However, I think decoupling that rigorously would make this harder to read, and also this is all going to get refactored shortly anyway.